docs/source/examples/training/index.rst

```diff
@@ -13,5 +13,6 @@ Training
    NeMo <nemo.md>
    Ray <ray.md>
    Training on TPUs <tpu.md>
+   Training with EFA <aws_efa.md>
```

Comment:
It seems we don't have training examples yet. How about calling it `AWS EFA` and move it to `Other Frameworks` instead?

Reply:
Updated